net/dns/direct.go

```diff
@@ -336,7 +337,7 @@ func (m *directManager) checkForFileTrample() {
 		show = show[:1024]
 	}
 	m.logf("trample: resolv.conf changed from what we expected. did some other program interfere? current contents: %q", show)
-	// TODO(bradfitz): register with health package that something is wrong
+	warnTrample.Set(errors.New("Linux DNS config not ideal. /etc/resolv.conf overwritten. See https://tailscale.com/s/dns-fight"))
```

Comment:
We'll use https://tailscale.com/s/* to redirect to KB articles?

Reply:
Yeah, we use /s/ already in a handful of places in the client, as we can't update old clients in the field but we can update our redirect.txt file :)